SConstruct

```diff
@@ -0,0 +1,3 @@
+# -*- python -*-
+from lsst.sconsUtils import scripts
+scripts.BasicSConstruct("spectractor", disableCc=True)
```

Comment:
Did you actually test that this builds the package, runs its tests, and makes it properly available?  I think the organization of directories (no `python` directory) and the lack of `tests/SConscript` is such that it will not.